src/vs/workbench/contrib/terminalContrib/suggest/browser/terminalSuggestAddon.ts

```diff
@@ -814,8 +814,9 @@ export class SuggestAddon extends Disposable implements ITerminalAddon, ISuggest
 		if (!suggestion) {
 			suggestion = this._suggestWidget?.getFocusedItem();
 		}
+
 		const initialPromptInputState = this._mostRecentPromptInputState;
-		if (!suggestion || !initialPromptInputState || this._leadingLineContent === undefined || !this._model) {
+		if (!suggestion?.item || !initialPromptInputState || this._leadingLineContent === undefined || !this._model) {
```

Comment:
This check looks fine, but shouldn't the command never be run to begin with due to context keys?

Reply:
oh, maybe this is fixed now that we don't automatically push the inline suggestion (earlier pr?)

Reply:
no, it's not